docs/en/connector-v2/source/Paimon.md

```diff
@@ -40,8 +53,37 @@ The table you want to access
 
 The file path of `hdfs-site.xml`
 
+### paimon.read.filter.sql [string]
+
+The filter condition of the table read. For example: `select * from st_test where id > 100`. If not specified, all rows are read.
+Currently, where conditions only support <, <=, >, >=, =, !=, or, and,is null, is not null, and others are not supported.
```

Comment:
'Only SELECT statements with WHERE clause are supported. The Having, Group By, Order By, Limit clauses are currently unsupported',these also need to be explained in the documentation

Reply:
done